intranet/apps/auth/views.py

```diff
@@ -103,6 +107,42 @@ def get_week_sports_school_events() -> Tuple[Container[Event], Container[Event]]
     return sports_events, school_events
 
 
+def lockout_view(request):
+    if request.method == "POST":
+        form = ResetLoginAttemptsEmailForm(request.POST)
+        if form.is_valid():
+            user = form.cleaned_data["user_account"]
+            subject = "Account Unlock Requested for {}".format(user)
+            emails = [settings.FEEDBACK_EMAIL]
+            base_url = "https://ion.tjhsst.edu/unlock"
+            data = {"user": user, "url": base_url}
+            email_send_task.delay("auth/emails/reset_login.txt", "auth/emails/reset_login.html", data, subject, emails)
+            return HttpResponseRedirect("/")
+        return render(request, "auth/lockout.html", context={"cooloff": settings.AXES_COOLOFF_TIME, "form": form, "error": True})
+    form = ResetLoginAttemptsEmailForm()
+    return render(request, "auth/lockout.html", context={"cooloff": settings.AXES_COOLOFF_TIME, "form": form, "error": False})
+
+
+@login_required
+@deny_restricted
+def unlock_view(request):
+    is_admin_all = request.user.has_admin_permission("all")
+    if not is_admin_all:
+        return render(request, "error/403.html", {"reason": "You are not authorized to view this page."}, status=403)
+    if request.method == "POST":
+        form = ResetLoginForm(request.POST)
+        if form.is_valid():
+            if form.cleaned_data["reset_all"]:
+                reset()
+            else:
+                reset(username=form.cleaned_data["user_account"])
+            return HttpResponseRedirect("/unlock")
+        return render(request, "auth/unlock.html", context={"form": form, "error": True})
```

Comment:
You don't need to reimplement error handling. Django natively supports this with form validators (see [docs on `Field.error_messages`](https://docs.djangoproject.com/en/3.0/ref/forms/fields/#django.forms.Field.error_messages))